src/test/java/ICS_Account_functionality.java

```diff
@@ -0,0 +1,115 @@
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.junit.Assert;
+import org.openqa.selenium.*;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class ICS_Account_functionality {
+    public static void main(String[] args) throws InterruptedException {
+        accountTest("desktop");
+        accountTest("tablet");
+        accountTest("mobile");
+    }
+
+    public static void accountTest(String size) throws InterruptedException {
+        WebDriverManager.chromedriver().setup();
+
+        WebDriver driver = driverSetup(size);
+
+        Dimension desktop = new Dimension(1366, 766);
+        Dimension tablet = new Dimension(766, 1024);
+        Dimension mobile = new Dimension(400, 800);
+
+        if (size.equals("desktop")) {
+            driver.manage().window().setSize(desktop);
+        } else if (size.equals("tablet")) {
+            driver.manage().window().setSize(tablet);
+        } else {
+            driver.manage().window().setSize(mobile);
+        }
+
+        driver.get("https://icecreamshop-frontend-rkbemzbp2q-ez.a.run.app/");
+
+        //check if "Account" button from Homepage it's written correctly
+        if (size.equals("desktop") || size.equals("tablet")) {
+            String accountButton = driver.findElement(By.cssSelector("[class='TopSection_accountLabel__1nFim']")).getText();
+            boolean condition1 = false;
+            if (accountButton.equals("Account")) {
+                condition1 = true;
+            }
+            Assert.assertTrue(condition1);
+            System.out.println(accountButton);
+
+            WebElement accountClickOnButton = driver.findElement(By.cssSelector("[class='TopSection_accountLabel__1nFim']"));
+            accountClickOnButton.click();
+            Thread.sleep(3000);
+
+        } else {
+            WebElement menuButton = driver.findElement(By.xpath("//*[@id=\"TopSection_main__2Uztx\"]/button"));
+            menuButton.click();
+            Thread.sleep(3000);
+
+            String accountButtonM = driver.findElement(By.cssSelector("[class='TopSection_btnMenuAccount___JCyj']")).getText();
+            boolean condition1 = false;
+            if (accountButtonM.equals("Account")) {
+                condition1 = true;
+            }
+            Assert.assertTrue(condition1);
+            System.out.println(accountButtonM);
+
+            WebElement accountClickOnButtonM = driver.findElement(By.cssSelector("[class='TopSection_btnMenuAccount___JCyj']"));
+            accountClickOnButtonM.click();
+            Thread.sleep(3000);
+
```

Comment:
instead of thread sleep you can use a better version : 

WebDriverWait wait = new WebDriverWait(driver, 30); // the integer can be set to a higher or lower value depending on the load time of the page
        wait.until(ExpectedConditions.**presenceOfElementLocated**(By.cssSelector(whatever_you_want_to_use)));

there is another condition also : element to be clickable